cypress/cypress/integration/home.spec.ts

```diff
@@ -110,6 +110,74 @@ describe("My Mentor Page", () => {
     cy.get("[data-cy=recommended-action-button]").should("exist");
   });
 
+  it("shows placeholder when no thumbnail", () => {
+    cySetup(cy);
+    const testClint = {
```

Comment:
inline all these `testClint` variables into the mock call. As a general rule, if something can be inlined (because nothing else in scope accesses it), then inline it